package.json

```diff
@@ -94,7 +94,7 @@
     "react-dropzone": "^7.0.1",
     "react-hot-loader": "^4.8.4",
     "react-offcanvas": "^0.4.0",
-    "react-redux": "^6.0.0",
+    "react-redux": "^7.1.1",
```

Comment:
Should this sort of change be in a separate PR?

Reply:
I'm not sure, I needed to upgrade to the `7.1.1` version to use the React hooks `useDispatch`, `useSelector` from this module.